src/types/user.ts

```diff
@@ -518,10 +518,10 @@ export default /* GraphQL */ `
     createdAt: DateTime!
 
     """ does this address own any Travelogger NFTs? this value is cached at most 1day, and refreshed at next \`nfts\` query """
-    hasNFTs: Boolean!
+    hasNFTs: Boolean! @objectCache(maxAge: ${CACHE_TTL.LONG})
 
     """NFT assets owned by this wallet address"""
-    nfts: [NFTAsset!]
+    nfts: [NFTAsset!] @cacheControl(maxAge: ${CACHE_TTL.MEDIUM}, scope: PRIVATE)
```

Comment:
1. I think it won't work since the query from matters.news includes [other fields](https://github.com/thematters/matters-web/blob/2c529b1bcb6101ae0960d930a114bc7583df09e2/src/components/UserProfile/DropdownActions/EditProfileDialog/gql.ts#L24-L35) that TTL are [lower](https://github.com/thematters/matters-server/blob/develop/src/types/directives/privateCache.ts#L26) than it, and apollo will cache as the lowest one.

> Important: Apollo Server assigns each GraphQL response a maxAge equal to the lowest maxAge among included fields. If any field has a maxAge of 0, the response will not be cached at all.
> 
> https://www.apollographql.com/docs/apollo-server/performance/caching/

2. Since `NFTAsset` doesn't include user-specific fields,  Is `scope: PRIVATE` not necessary?